static/js/components/EducationDialog.js

```diff
@@ -8,21 +9,22 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import { educationValidation } from '../util/validation';
 
 export default class EducationDialog extends ProfileFormFields {
-  constructor(props) {
+  constructor(props: {}) {
```

Comment:
my understanding is that `number` refers to anything declare with a literal (e.g. `var foo: number = 2`) whereas `Number` refers to things explicitly wrapped with the `Number` constructor (e.g. `var foo: Number = new Number(2)`)